runtime/services/lightd/effects.js

```diff
@@ -87,6 +88,12 @@ module.exports = LightRenderingContextManager
  */
 function LightRenderingContextManager () {
   this.id = 0
+  this.ledsConfig = light.getProfile()
+  var ledsConfig = light.getProfile()
```

Comment:
Why do you get twice in different variables?

Reply:
Sorry, updated.